atd-vze/src/views/Crashes/Crash.js

```diff
@@ -55,6 +55,15 @@ const calculateYearsLifeLost = people => {
 };
 
 function Crash(props) {
+  const [isCOA, setIsCOA] = useState(false);
+  useEffect(() => {
+    if (user && user.email) {
+      if (user.email.toLowerCase().endsWith("@austintexas.gov")) {
+        setIsCOA(true);
+      }
+    }
+  }, [user]);
```

Comment:
Another thank you to @mddilley for catching my omission of `user` from the dependency array. 🙏